src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -1278,8 +1289,8 @@ def service_name(self):
         if self.daemon_type == 'rgw':
             v = self.daemon_id.split('.')
             s_name = '.'.join(v[0:2])
-            return 'rgw.%s' % s_name
-        if self.daemon_type in ['mds', 'nfs']:
+            return '$s.%s' % (self.daemon_type, s_name)
```

Comment:
https://jenkins.ceph.com/job/ceph-pull-requests/48216/consoleFull#1872527456e840cee4-f4a4-4183-81dd-42855615f2c1

```
orchestrator/_interface.py: note: In member "service_name" of class "DaemonDescription":
orchestrator/_interface.py:1292: error: Not all arguments converted during string formatting
```

```suggestion
            return '%s.%s' % (self.daemon_type, s_name)
```